prep-css-flexbox/justify-content.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    #card-deck {
+      display: flex;
+      padding: 10px;
+    }
+
+    .card-container {
+      flex-basis: 25%;
+      justify-content: center;
```

Comment:
`justify-content` should be on the flex container. So in this case `justify-content` should be applied to the `#card-deck` element.